src/appmetrics.cpp

```diff
@@ -388,10 +397,22 @@ static void sendData(const std::string &sourceId, unsigned int size, void *data)
 	memcpy(dataCopy, data, size);
 	payload->data = dataCopy;
 	payload->size = size;
-
-	async->data = payload;
-	uv_async_init(uv_default_loop(), async, (uv_async_cb)emitMessage);
```

Comment:
I'll remove that, we aren't just declaring it but new'ing a new one and leaking memory. (Good spot.)